.ci/pipeline/dr_weekly_jjb.yaml

```diff
@@ -0,0 +1,68 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: pipeline
+    folder: pdc
+    properties:
+        - github:
+            url: "https://github.com/Mellanox/libvma"
+        - build-discarder:
+            days-to-keep: 50
+            num-to-keep: 20
+        - inject:
+            keep-system-variables: true
+            properties-content: |
+              jjb_proj={jjb_proj}
+    description: Do NOT edit this job through the Web GUI !
+    concurrent: false
+    parameters:
+        - string: 
+            name: "sha1"
+            default: "master"
+            description: "Branch or sha1 to check out"
+        - string:
+            name: "release_tag"
+            default: ""
+            description: "Tag to release"
+        - string:
+            name: "release_folder"
+            default: "/tmp"
+            description: "Folder to release packages into. For NFS tests use /auto/mswg/release/vma/dr/weekly"
+        - string:
+            name: "CONF_FILE"
+            default: ".ci/pipeline/dr_weekly_matrix.yaml"
+            description: "Regex to select job config file"
+        - string:
+            name: "MAIL_TO"
+            default: "c2a304c7.NVIDIA.onmicrosoft.com@amer.teams.ms"
```

Comment:
Please add description of what is this teams address stands for

Reply:
> Please add description of what is this teams address stands for

Added the comment with the description, [line-36](https://github.com/Mellanox/libvma/commit/51cd8c67361ea2710f09260868fd3dcf2b49eeca#diff-ca0678ddee95e31f8735a639318fb34ce7d730ac7772e9e3b65708f33ec3e28bR36)